index.html

```diff
@@ -5,15 +5,58 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <title>Karma</title>
     <link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
```

Comment:
Hi Yulia,
please clean up unnecessary font imports 
there is reference to google font above:
```html
<link href='https://fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
```
this tag links with an external css where all the imports are being done for you. 
the rest if links to any other fonts you don't really need:
```hml
<link rel="preconnect" href="https://fonts.googleapis.com">
<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap" rel="stylesheet">
```
remove them as well as css import
```html
<link rel="stylesheet" href="./css/fonts.css">
```
and `./css/fonts.css` file as well